src/probe_types.cpp

```diff
@@ -30,6 +30,35 @@ ProbeType probetype(const std::string &probeName)
   return retType;
 }
 
+enum bpf_prog_type get_bpf_prog_type(ProbeType type)
```

Comment:
This function already exists in `attached_probe.cpp` - can we consolidate and use just a single helper?